docs/tutorial/file.rst

```diff
@@ -102,32 +121,25 @@ All channel data can be accessed using the above method. High frequency force da
     file.force1x
 
 The only difference between the two above methods for accessing channel data, is that `file.force1x` allows you to access the force calibration data, as described below. 
-The channels have a few convenient methods, like `.plot()` which make it easy to preview the contents::
+Plotting the data can be done as follows::
 
     file.force1x.plot()
     plt.savefig("force1x.png")
 
-You can also always access the raw data directly::
+You can also access the raw data directly::
 
     f1x_data = file.force1x.data
     f1x_timestamps = file.force1x.timestamps
     plt.plot(f1x_timestamps, f1x_data)
 
 The `timestamps` attribute returns the measurement time in nanoseconds since epoch (January 1st 1970, midnight UTC/GMT).
-Note that since these values are typically very large, they cannot be converted to floating point without losing precision::
 
-    >>> t = f1x_timestamps[0]
-    >>> roundtrip_t = np.int64(np.float64(t))
-    >>> print(t - roundtrip_t)
-    24
-
-The reason for this is that timestamps exceed the maximum integer value representable by the mantissa.
 The relative time values in seconds can also be accessed directly::
 
```

Comment:
Did you remove the warning about the lossiness of converting timestamps to/from floats to try and not distract from the main line of the story?

Would it be possible to add it back somewhere else? Or devote a sentence that care must be taken that these timestamps cannot be converted to floating point without losing some precision (maybe without the explicit example so that it doesn't distract too much)? Reason being that we have seen this go wrong a few times.

Reply:
didn't realize that this was actually an error that pops up regularly. Put it back, with the example.

Reply:
👍